src/templates/games/lobby_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h2>Create a new Lobby</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button type="submit">Save</button>
+  </form>
+{% endblock body %}
```

Comment:
I'm getting a syntax error, but it's fixed by changing this line to `endblock content` or simply `endblock`

Reply:
thank you! fixed